encoder/vaapiencoder_base.cpp

```diff
@@ -440,16 +440,22 @@ struct ProfileMapItem {
     VAProfile    vaProfile;
 };
 
+
+
 const ProfileMapItem g_profileMap[] = {
     { PROFILE_H264_BASELINE, VAProfileH264Baseline },
     { PROFILE_H264_CONSTRAINED_BASELINE, VAProfileH264ConstrainedBaseline },
     { PROFILE_H264_MAIN, VAProfileH264Main },
     { PROFILE_H264_HIGH, VAProfileH264High },
     { PROFILE_JPEG_BASELINE, VAProfileJPEGBaseline },
-    { PROFILE_H265_MAIN, VAProfileHEVCMain },
-    { PROFILE_H265_MAIN10, VAProfileHEVCMain10 },
+    #if VA_CHECK_VERSION(0,38,0)
```

Comment:
is it libva version  0.37 enough? I see AS_IF([test x$enable_h265dec = xyes], [VA_API_VERSION="0.37"]) in the configure.ac.